### PR TITLE
Modify the order of Privacy Policy and About in Settings

### PR DESCRIPTION
**Pull request by afr30010**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

**Comment by sarentz** [comment]
@afr30010 Can you please explain what this pull request does in the first comment?

**Comment by afr30010** [comment]
This pull request simply modify the order of these 2 elements.

**The order before this commit was:** 
1. Help
2. Rate on Google Play
3. About Firefox Preview
4. Privacy Policy

**After this commit is:**
1. Help
2. Rate on Google Play
3. Privacy Policy
4. About Firefox Preview

IMO, It looks better than before with this change.

**Comment by sarentz** [comment]
What do you think @topotropic - I kind of agree - i had to look up the version number yesterday and my fingers expected to find About Firefox Preview at the bottom.

**Comment by afr30010** [comment]
@sarentz Rebased for the latest changes.

**Comment by mheubusch** [comment]
@st3fan A few things: The label should be "Privacy notice" and the n is lower case. It should also appear as the last item in the  Privacy subsection  (which is still called Advanced) - under Data collection (currently says Data choices and has  the wrong info on the Data collection subpanel

**Comment by sblatz** [comment]
@afr30010 Can you please make the "n" lowercase in "privacy notice"? See [@mheubusch's comment](https://github.com/mozilla-mobile/fenix/pull/2760#issuecomment-497815231)

You'll also want to fix these merge conflicts. After that, @ mention me and I can get this merged in!

**Comment by afr30010** [comment]
@sblatz

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -133,6 +131,8 @@
     <string name="preferences_language">Language</string>
     <!-- Preference for data choices -->
     <string name="preferences_data_choices">Data choices</string>
+    <!-- Preference linking to the privacy notice -->
+    <string name="preferences_privacy_notice">Privacy notice</string>
```
Comment (sblatz) [inline app/src/main/res/values/strings.xml]: Sorry, one more nit here: we will actually need to change the string key `preferences_privacy_notice` to something else so our translators see that a change was made. Perhaps something like `preferences_privacy_link`

Comment (afr30010) [reply app/src/main/res/values/strings.xml]: Should I also change the label to "Privacy link"?

Comment (sblatz) [reply app/src/main/res/values/strings.xml]: The string should stay the same, just need the "name" portion to change :)

Comment (afr30010) [reply app/src/main/res/values/strings.xml]: Done.

**Comment by sblatz** [comment]
@afr30010 can you pull (upstream) master --rebase and push again? :)

**Comment by afr30010** [comment]
> @afr30010 can you pull (upstream) master --rebase and push again? :)

Done.

**Comment by sblatz** [comment]
@afr30010 There are build failures. I think we just need to update the places that use the older pref key we changed :)
image

**Pull request merged**